pom.xml

```diff
@@ -3,12 +3,12 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532</version>
+    <version>1.542</version>
   </parent>
 
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>diskcheck</artifactId>
-  <version>0.27</version>
+  <version>0.28</version>
```

Comment:
You shouldn't touch that field in a PR generally. 
Btw, it's not normal it doesn't end with -SNAPSHOT.

-SNAPSHOT should only be absent in tags.